test/general/implementation_details/balanced_path_unit_tests.pass.cpp

```diff
@@ -178,7 +178,7 @@ test_serial_set_op_count_and_write2_large_setA(SetTag set_tag)
     std::vector<int> v2 = {1};
     std::vector<int> v3(v1.size() + v2.size());
 
-    oneapi::dpl::__par_backend_hetero::__temp_data_array<10, int> __temp_data{};
+    oneapi::dpl::__par_backend_hetero::__temp_data_array<15, int> __temp_data{};
```

Comment:
Probably we can define the amount of items in `v2` array through new compile-time const.